src/features/removeRedirection.ts

```diff
@@ -29,4 +30,16 @@ export const maybeRemoveRedirection = makeBTDModule(({TD, settings}) => {
 
     return result;
   };
+
+  TD.services.TwitterUser.prototype.getExpandedURL = function () {
+    return this.entities.url.urls[0].expanded_url;
```

Comment:
Could we add some checks here? I feel like this might crash if a given user doesn't have an url?

Reply:
As far as the code in this PR, `getExpandedURL()` is not called for users who do not have a website link so that no crash will occur. 
However, with this, templates such as `{{# getExpandedURL}} ... {{/ getExpandedURL}}` cause crash, now I think we should add an existence check too.

So, I'll add the check similar to `TD.services.TwitterUser.prototype.getDisplayURL` (implemented by original TweetDeck)

Reply:
@hiroto7 sounds good! I prefer to be extra careful because crashing in there can have weird consequences 😅

Reply:
I added it in 84ead19.